openff/toolkit/tests/test_molecule.py

```diff
@@ -3218,8 +3103,8 @@ def test_divalent_lone_pair_virtual_site_position(self):
         atom3 = molecule.atoms[2]
         distance = 0.2 * unit.angstrom
         out_of_plane_angle = 90 * unit.degree
-        vsite1_index = molecule.add_divalent_lone_pair_virtual_site(
-            [atom1, atom2, atom3],
+        molecule.add_divalent_lone_pair_virtual_site(
+            [atom2, atom1, atom3],
```

Comment:
Divalent atom ordering now matches SMIRNOFF spec, so need to update the tests which still use the incorrect order